client/src/components/home/stats.tsx

```diff
@@ -1,21 +1,43 @@
 import { cn } from '@/lib/classnames';
 
-import { getHomeStats } from '@/types/generated/home-stat';
+import { getPractices } from '@/types/generated/practice';
+import { getProjects } from '@/types/generated/project';
+
+import { getDatasets } from '@/hooks/datasets';
+import { getScientificEvidenceMockStats } from '@/hooks/scientific-evidence';
 
 import Counter from './counter';
 
 export default async function Stats({ className }: { className: string }) {
-  const data = await getHomeStats({ populate: '*', sort: 'id' });
-  const stats = data?.data?.map((item) => ({
-    title: item?.attributes?.title,
-    value: item?.attributes?.value,
-    class: item?.attributes?.class,
-  }));
-  if (!data || !stats) return null;
+  const data: { title: string; class: string; value?: number }[] = [
+    {
+      title: 'Meta-analysis',
+      value: (await getScientificEvidenceMockStats()).all?.metaAnalysis,
+      class: 'bg-green-700',
+    },
+    {
+      title: 'Initiatives',
+      value: (await getProjects({ fields: ['id'], 'pagination[pageSize]': 1 })).meta?.pagination
+        ?.total,
+      class: 'bg-blue-500',
+    },
+    {
+      title: 'Datasets',
+      value: (await getDatasets({ size: 1 })).meta?.total_records,
+      class: 'bg-purple-700',
+    },
+    {
+      title: 'Practices',
+      value: (await getPractices({ fields: ['id'], 'pagination[pageSize]': 1 })).meta?.pagination
+        ?.total,
+      class: 'bg-brown-500',
+    },
+  ];
+  if (!data) return null;
```

Comment:
I don't think this condition is useful anymore because the array will never be falsy.